packages/compiler/src/formatter/print/printer.ts

```diff
@@ -1920,7 +1920,7 @@ export function printStringTemplateExpression(
   }
 }
 
-function splitLines(text: string): string[] {
+export function splitLines(text: string): string[] {
```

Comment:
i think this should be moved out of the printer. Check `syntax-utils` where we moved some other printing related utils.

Reply:
as the other comment also found, seems like there is a bug, it shouldn't include the new lines. I think it kinda works for the formatter use because it counts the newline in the `whitespaceIndent` but if you fix the function it work as well.